aptos-move/aptos-vm/src/aptos_vm_impl.rs

```diff
@@ -43,10 +43,9 @@ use std::sync::Arc;
 
 pub const MAXIMUM_APPROVED_TRANSACTION_SIZE: u64 = 1024 * 1024;
 
-#[derive(Clone)]
-/// A wrapper to make VMRuntime standalone and thread safe.
+/// A wrapper to make VMRuntime standalone
 pub struct AptosVMImpl {
-    move_vm: Arc<MoveVmExt>,
+    move_vm: MoveVmExt,
```

Comment:
cc: @gelash I think we may want to bring the Arc back later when the new loader can be shared across threads